spec/acceptance/policy_spec.rb

```diff
@@ -21,14 +21,16 @@ class { '::rabbitmq':
         ensure => present,
       } ->
 
-      rabbitmq_policy { 'ha-all@myhost':
-        pattern    => '.*',
-        priority   => 0,
+      rabbitmq_policy { 'ha-exactly@myhost':
+        pattern    => 'ha_.*',
+        priority   => 1,
         applyto    => 'all',
-        definition => {
-          'ha-mode'      => 'all',
-          'ha-sync-mode' => 'automatic',
-        },
+        # Workaround for Puppet interpreting numbers as strings
```

Comment:
@hunner

I'm unsure about munging certain strings to integers if Puppet 4.0.0 will fix it.  What if RabbitMQ decides to introduce a policy definition where the value can match `^\d+$` but is supposed to be a string?  Then we'll reach a similar issue to what we have now (now, if I specify an integer for ha-params, it fails because of the validation; if I specify a string, then it fails because `rabbitmqctl set_policy` fails.